### PR TITLE
Added missing config key and removed unnecessary imports.

### PR DESCRIPTION
**Pull request by dribnet**
Added n_classes to IAN config, which is needed by get_model.
This allows the sample_IAN to launch as given in the README.

Also removed some unnecessary imports from sample_IAN
(voxnet and CAcheckpoints).

### CODE REVIEW

**Comment by ajbrock** [comment]
Hi Tom,

Thanks for catching those--I'm actually about to upload one pre-trained IAN model with a slightly different config and architecture (there's an extra set of batch norms in the one I put up, which is what happens when you upload things after extended sleepless periods)--so this may be fixed shortly. Much appreciated, though, I always forget to pull those two imports when I make new repos.
